gsa/src/web/pages/page.js

```diff
@@ -116,9 +114,9 @@ const Page = props => {
 };
 
 Page.propTypes = {
-  gmp: PropTypes.gmp.isRequired,
+  children: PropTypes.func.isRequired,
 };
 
-export default compose(withGmp, withRouter)(Page);
+export default withRouter(Page);
```

Comment:
withRouter can be dropped when using useLocation
```suggestion
export default Page;
```